projects/laji/src/app/shared/service/laji-api.service.ts

```diff
@@ -278,15 +278,15 @@ export class LajiApiService {
     );
   }
 
-  private removeUndefinedFromObject(obj: object) {
+  private removeUndefinedFromObject<T extends object>(obj: T): T {
```

Comment:
Doesn't this line contradict the next one? If T extends object, then there should be no need to check whether obj is an object or not?